packages/cubejs-testing/test/events-only-test-case.ts

```diff
@@ -0,0 +1,265 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { jest, expect, beforeAll, afterAll } from '@jest/globals';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import cubejs, { Query, CubejsApi } from '@cubejs-client/core';
+import WebSocketTransport from '@cubejs-client/ws-transport';
+import { BirdBox } from '../src';
+
+type QueryTestOptions = {
+  name: string;
+  ws?: true,
+};
+
+const asserts: [options: QueryTestOptions, query: Query][] = [
+  [
+    {
+      name: '#1 Events.count',
+      ws: true,
+    },
+    {
+      measures: [
+        'Events.count'
+      ],
+      timeDimensions: [],
+      order: {},
+      dimensions: []
+    },
+  ],
+  [
+    {
+      name: '#2 Events.count with Events.type order by Events.count DESC',
+      ws: true,
+    },
+    {
+      measures: [
+        'Events.count'
+      ],
+      timeDimensions: [],
+      order: {
+        'Events.count': 'desc'
+      },
+      dimensions: [
+        'Events.type'
+      ]
+    }
+  ],
+];
+
+// eslint-disable-next-line import/prefer-default-export
+export function createBirdBoxTestCase(name: string, entrypoint: () => Promise<BirdBox>) {
+  describe(name, () => {
+    jest.setTimeout(60 * 5 * 1000);
+
+    let birdbox: BirdBox;
+    let wsTransport: WebSocketTransport;
+    let httpClient: CubejsApi;
+    let wsClient: CubejsApi;
+
+    // eslint-disable-next-line consistent-return
```

Comment:
Since you are creating this file from scratch, I propose drastic simplifications:

1. Run only httpClient tests. Originally these are `#1 Orders.totalAmount`, `#2 Events.count`, `#3 Events.count with Events.type order by Events.count DESC` and `Dbt orders count`.
2. Define the tests inline, `it(<name>, async () => <body>)`, as opposed to indirecting throught the `asserts` table. 
3. For the Orders test, create a new OrdersQDB.js schema file, to account for the Questdb quirks when it comes to SELECT UNION SELECT queries.
4. Skip the DBT test.